web-app/src/assets/i18n/ja-JP.json

```
@@ -0,0 +1,850 @@
+  "alert.setting.template.example": "通知テンプレートを入力してください。例：${app}.${metrics}.${metric}の値が高すぎます",
+  "alert.setting.template.label": "アラームがトリガーされた後に送信される通知情報のテンプレート。上記のテンプレート環境変数を参照してください。",
+  "alert.setting.template.metric-name": "メトリック名",
+  "alert.setting.template.metric-value": "メトリック値",
+  "alert.setting.template.metrics-name": "メトリック名",
+  "alert.setting.template.monitor-type": "モニタータイプ名",
+  "alert.setting.template.other-value": "他のメトリック値",
+  "alert.setting.template.tip": "サポートされている通知テンプレート環境変数",
+  "alert.setting.template.vars.app": "アプリケーション名",
+  "alert.setting.template.vars.instance": "インスタンスID",
+  "alert.setting.template.vars.metrics": "メトリック名",
+  "alert.setting.template.vars.threshold": "閾値式",
+  "alert.setting.template.vars.time": "トリガー時間",
+  "alert.setting.template.vars.tip": "メトリックや演算子を挿入",
+  "alert.setting.template.vars.value": "トリガー値",
+  "alert.setting.time": "時間",
+  "alert.setting.times": "トリガー回数",
+  "alert.setting.times.tip": "アラームを送信する前に閾値が何回トリガーされるかを設定",
+  "alert.setting.trigger": "アラームをトリガー",
+  "alert.setting.type": "閾値タイプ",
+  "alert.setting.type.periodic": "周期的",
+  "alert.setting.type.periodic.desc": "PromQLクエリを定期的に実行して閾値アラートをトリガーします。",
+  "alert.setting.type.realtime": "リアルタイム",
+  "alert.setting.type.realtime.desc": "リアルタイムメトリック計算と閾値超過時の即時アラート。",
+  "alert.severity": "アラームの重大度",
+  "alert.severity.0": "緊急",
+  "alert.severity.1": "クリティカル",
+  "alert.severity.2": "警告",
+  "alert.severity.all": "すべての重大度",
+  "alert.silence.delete": "サイレンス戦略を削除",
+  "alert.silence.edit": "サイレンス戦略を編集",
+  "alert.silence.enable": "サイレンスを有効化",
+  "alert.silence.labels": "タグ一致",
+  "alert.silence.match-all": "すべて一致",
+  "alert.silence.name": "サイレンス戦略名",
+  "alert.silence.new": "新しいサイレンス戦略",
+  "alert.silence.priority": "優先度一致",
+  "alert.silence.time": "サイレンス期間",
+  "alert.silence.times": "サイレンスされたアラート数",
+  "alert.silence.type": "サイレンスタイプ",
+  "alert.silence.type.cyc": "周期的サイレンス",
+  "alert.silence.type.once": "一時的サイレンス",
+  "alert.status": "アラートステータス",
+  "alert.status.all": "すべてのステータス",
+  "alert.status.firing": "発火中",
+  "alert.status.resolved": "解決済み",
+  "app.lock": "ロック解除",
+  "app.lock.placeholder": "解除するには何かを入力してください",
+  "app.login.explore.cloud": "クラウドを探索",
+  "app.login.explore.cloud.detail": "HertzBeatクラウドを探索するにはクリック",
+  "app.login.login": "ログイン",
+  "app.login.message-invalid-credentials": "無効なユーザー名またはパスワード",
+  "app.login.message-need-credential": "パスワードを入力してください",
+  "app.login.message-need-identifier": "ユーザー名を入力してください",
+  "app.login.need-change-password": "初期デフォルトパスワードを適時に更新してください！",
+  "app.login.notify": "ログインしてください！",
+  "app.login.remember-me": "ログイン状態を保持",
+  "app.login.tab-login-credentials": "HertzBeatにサインイン",
+  "app.passport.desc": "オープンソースのリアルタイム監視システム",
+  "app.passport.intro-1": "オープンソース、分散型",
+  "app.passport.intro-2": "リアルタイム監視",
+  "app.role.admin": "管理者",
+  "app.theme.compact": "コンパクトテーマ",
+  "app.theme.dark": "ダークテーマ",
+  "app.theme.default": "ライトテーマ",
+  "bulletin.batch.delete": "バレットンを一括削除",
```

```suggestion
  "bulletin.batch.delete": "速報を一括削除",
```
modify バレットン to 速報, same below